docs/hub/security-pickle.md

```diff
@@ -0,0 +1,226 @@
+# Pickle Scanning
+
+## What is a pickle ?
```

Comment:
Maybe we could have a summary section before this

Reply:
Is what I added ok or do you want something a bit lengthier ?